SETUP/devex/dpdev-docker/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:8.1-apache
+
+# Install dependency packages
+RUN apt-get update
+# DP deps: mariadb-client gettext aspell aspell-fr pngcheck
+# Handy dev deps: unzip git
+RUN apt-get install -y mariadb-client gettext aspell aspell-fr pngcheck git unzip
```

Comment:
reading https://docs.docker.com/build/building/best-practices/#run, it seems we should combine the update in this command as well as a few other tweaks:
```
RUN apt-get update &&  apt-get install -y --no-install-recommends \
mariadb-client \
gettext \
aspell \
aspell-fr \
pngcheck \
git \
unzip \
&& rm -rf /var/lib/apt/lists/*
```

Reply:
Clearing the apt cache can be important in small images, however this isn't one. My build clocks in a bit under 1GB, whereas cleaning up the apt-cache would trim off only 19MB. There's no reason to **not** do this, but it isn't a significant difference IMHO.

Reply:
To find out, I looked at the install as-is vs. adding `--no-install-recommends` and the difference seems to be about 24MB.

Reply:
It's a good suggestion but I'm going to punt it for now. Note also that `install-php-extension` is a whole script that also does a `apt-get update` and then removes the caches (and itself) at the end of the run. So there's a few things going on here that could be optimized.

PRs to further optimize the Dockerfile after this merges in are welcome!